.github/workflows/gnoland.yml

```diff
@@ -101,7 +99,7 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           directory: ${{ runner.temp }}/coverage
-          token: ${{ secrets.CODECOV_TOKEN }}
+          token: ${{ secrets.CODECOV_TOKEN }} 
```

Comment:
Shouldn't we add an `.editorconfig` file to enforce the removal of trailing whitespaces in most IDEs?

Reply:
this is is side effect of the reverted commits, and yes I agree we should have an `.editorconfig`

Reply:
You know what to do :)